pkg/database/database.go

```diff
@@ -42,31 +33,25 @@ func InitDb() {
 }
 
 func dbConfig() map[string]string {
-	conf := make(map[string]string)
-	host, ok := os.LookupEnv(dbhost)
-	if !ok {
-		panic("DBHOST environment variable required but not set")
-	}
-	port, ok := os.LookupEnv(dbport)
-	if !ok {
-		panic("DBPORT environment variable required but not set")
-	}
-	user, ok := os.LookupEnv(dbuser)
-	if !ok {
-		panic("DBUSER environment variable required but not set")
-	}
-	password, ok := os.LookupEnv(dbpass)
-	if !ok {
-		panic("DBPASS environment variable required but not set")
-	}
-	name, ok := os.LookupEnv(dbname)
-	if !ok {
-		panic("DBNAME environment variable required but not set")
+	viper.AddConfigPath("/Users/jordanlange/Documents/projects/personal/backlog-api/pkg/config")
```

Comment:
This should default to some sort of local file. Ideally also be configurable, but not required for now.

Reply:
Yeah agreed. I can just make it a relative path if that's better?